src/Response/StoreWebhookResponse.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NodelessIO\Response;
+
+class StoreWebhookResponse extends AbstractResponse
+{
+    public function getId(): string
+    {
+        return $this->getData()['id'];
+    }
+
+    public function getUrl()
+    {
+        return $this->getData()['url'];
+    }
+
+    public function getSecret(): float
```

Comment:
Does it make sense to return a secret? Is this bad security practice?

Reply:
Also it would be a string

Reply:
It makes only sense if the create webhook endpoint can be called without the secret as parameter and in that case would generate one. It will then only be returned once on creation and not on GET requests. This is how it works in BTCPay. Should I remove it or do you want to support the use case where no secret is passed on the POST request?

re "float", yes that's wrong, good catch